packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/find_shared_origin_objects.test.ts

```diff
@@ -10,6 +10,8 @@ import type { DeeplyMockedKeys } from '@kbn/utility-types-jest';
 import type { CreatePointInTimeFinderFn, PointInTimeFinder } from './point_in_time_finder';
 import { savedObjectsPointInTimeFinderMock } from '../mocks/point_in_time_finder.mock';
 import { findSharedOriginObjects } from './find_shared_origin_objects';
+import { ISavedObjectsRepository } from '@kbn/core-saved-objects-api-server';
+import { savedObjectsRepositoryMock } from '@kbn/core-saved-objects-api-server-mocks';
```

Comment:
I am looking into this. Joe had made this change before I took over work on this, so I am not entirely sure what the reason is yet. So far I cannot substitute `savedObjectsPointInTimeFinderMock.createClient` and get the test suite to pass. It may just be a mock problem, but on the other hand there may have been a good reason.

Reply:
It was just a missing mock resolved value for the openPointInTimeForType function. 👍